src/api/procedures/types.ts

```diff
@@ -1841,3 +1843,61 @@ export interface UpdatePolyxBondParams {
 export interface NominateValidatorsParams {
   validators: (Account | string)[];
 }
+
+export interface CreateBallotParams {
+  /**
+   * Title and motions of the Ballot
+   */
+  meta: BallotMeta;
+
+  /**
+   * Date when Ballot voting starts
+   */
+  startDate: Date;
+
+  /**
+   * Date when Ballot voting ends
+   */
+  endDate: Date;
+
+  /**
+   * Description of the Corporate Action to which the Ballot is attached
+   */
+  description: string;
+
+  /**
+   * Asset Holder Identities to be included (or excluded) from the Ballot. Inclusion/exclusion is controlled by the `treatment`
+   *   property. When the value is `Include`, all Asset Holders not present in the array are excluded, and vice-versa. If no value is passed,
+   *   the default value for the Asset is used. If there is no default value, all Asset Holders will be part of the Ballot
+   */
+  targets?: InputCorporateActionTargets;
+
+  /**
+   * Date on which the Corporate Action is declared
+   */
+  declarationDate: Date;
+
+  /**
+   * Specifies whether RCV is enabled for this ballot.
```

Comment:
I think expanding the acronym would be nice here

`Specifies whether Ranked Choice Voting (RCV) is enabled for this ballot.`